backend/src/test/lightning.local.e2e.test.ts

```diff
@@ -1,13 +1,19 @@
 import { Lightning } from '@inco/js/lite';
+import { Hex } from 'viem';
 import { anvil } from 'viem/chains';
 import { describe } from 'vitest';
-import { runE2ETest } from './lightning-test.ts';
+import { runE2ETest, runLibTestE2ETest } from './lightning-test.ts';
 
 describe(`Lightning Local Node E2E`, { timeout: 50_000 }, async () => {
-  const zap = Lightning.localNode();
+  const zap = await Lightning.localNode('alphanet');
   runE2ETest(Math.floor(Math.random() * 100), zap, {
     chain: anvil,
-    senderPrivKey: zap.deployment.senderPrivateKey,
+    senderPrivKey: zap.deployment.senderPrivateKey as Hex,
+    hostChainRpcUrl: 'http://127.0.0.1:8545',
+  });
+  runLibTestE2ETest(zap, {
+    chain: anvil,
+    senderPrivKey: zap.deployment.senderPrivateKey as Hex,
```

Comment:
let's break things down in a similar manner as we have them in monorepo for consistency